src/Stage/Parse/Parser.elm

```diff
@@ -885,3 +1177,12 @@ oneOrMoreHelp spaces p vs =
         , P.succeed ()
             |> P.map (always (P.Done (List.reverse vs)))
         ]
+
+
+{-| Taken from dmy/elm-pratt-parser, made to accept the operator parser result
+-}
```

Comment:
Can you expand on this description? Eg. in the `foo as bar` example, I'd imagine `as` being the operator, but it seems like you're saying `bar` is the operator parser result?